src/WingetCreateTests/WingetCreateTests/UnitTests/UpdateCommandTests.cs

```
@@ -82,7 +82,7 @@ public async Task UpdateCommandGitHubManifestTest()
         /// Tests the <see cref="UpdateCommand.DeserializeExistingManifestsAndUpdate"/> command, ensuring that it updates properties as expected.
         /// </summary>
         /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
-        //[Test]

```

Can you try uncommenting this and see if it still repros? I suspect it's intermittent, and instead of commenting out the test we should let it fail to give us pain to fix it

---

commented it out, lets see what happens

---

Looks like it passed